hop/auth.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+__author__ = "Patrick Godwin (patrick.godwin@psu.edu)"
+__description__ = "a module for auth utilities"
+
+import argparse
+import errno
+import logging
+import os
+
+from adc import auth
+
+DEFAULT_AUTH_CONFIG = """\
+security.protocol=SASL_SSL
+sasl.mechanism=PLAIN
```

Comment:
We should default to `SCRAM-SHA-512` here.

Reply:
Sounds good to me.